src/experimental/players/DesktopFloatOnScrollArticleVideoPlayer/DesktopFloatOnScrollArticleVideoPlayer.module.css

```diff
@@ -0,0 +1,25 @@
+.desktopArticleVideoTopPlaceholder {
+	position: absolute;
+	width: 100%;
+	padding-top: 56.25%;
+	top: 0;
+	left: 0;
+	bottom: 0;
+	right: 0;
+	z-index: 2;
+}
+
+.desktopArticleVideoWrapper {
+	height: max-content;
+	position: absolute;
+	bottom: 0;
+	right: 0;
+	top: 0;
+	left: 0;
+}
+
+.topBar {
+	width: 100%;
+	position: relative;
+	background: blue;
```

Comment:
Was it used only for debugging? The `TopBar` [styled component didn't have a blue background](https://github.com/Wikia/jwplayer-fandom/pull/166/files#diff-7363a5c12148ce18256b93461e5185c6660764b65f2429044605db50c78b005bL41-L44) 🤔

Reply:
Nicely caught! I've just removed that leftover.